docs/framework/winforms/high-dpi-support-in-windows-forms.md

```diff
@@ -36,42 +36,43 @@ The new Windows Forms features that support high DPI awareness are available onl
 
 In addition, to configure high DPI support in your Windows Forms application, you must do the following:
 
-- Declare compatibility with Windows 10. 
+- Declare compatibility with Windows 10.
 
    To do this, add the following to your manifest file:
 
    ```xml
    <compatibility xmlns="urn:schemas-microsoft.comn:compatibility.v1">
 	   <application>
-		   \<!-- Windows 10 compatibility -->
+		   <!-- Windows 10 compatibility -->
 		   <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
 	   </application>
    </compatibility>
    ```
-- Enable per-monitor DPI awareness in the app.config file.
+
+- Enable per-monitor DPI awareness in the *app.config* file.
 
    Windows Forms introduces a new [\<System.Windows.Forms.ApplicationConfigurationSection>]
-(Windows%20Forms%20Configuration%20Section.md) to support new features and customizations added starting with the .NET Framework 4.7. To take advantage of the new features that support high DPI, add the following to your application configuration file.   
+(../../../docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md) to support new features and customizations added starting with the .NET Framework 4.7. To take advantage of the new features that support high DPI, add the following to your application configuration file.
```

Comment:
instead of windows-forms-add-configuration-element.md, the link should go to its parent, index.md